rules/base.js

```diff
@@ -46,7 +46,7 @@ module.exports = {
 		'accessor-pairs': 2, // enforce getter and setter pairs in objects
 		'array-callback-return': 0, // enforce `return` statements in callbacks of array methods
 		'block-scoped-var': 0, // enforce the use of variables within the scope they are defined
-		'class-methods-use-this': 1, // enforce that class methods utilize `this`
+		'class-methods-use-this': 0, // don't enforce that class methods utilize `this`
```

Comment:
LOL I’m genuinely sorry to do this to you on such a small PR, buuuuut… can you keep the description as-is? The descriptions are copied verbatim from the ESLint docs, they aren’t supposed to reflect the current settting of the rule.

Sorry! I’m not trying to torture you, truly.

Reply:
No, that's fine! I wanna get it right. I'll submit again. 😄